examples/cdn-preact/README.md

```diff
@@ -0,0 +1,12 @@
+## Example: Consuming assets from the CDN
+
+This shows the usage of CMS design system preact components from the CDN:
+
+- Include the core CSS bundle and theme
+- Include the JS vendor files and design-system bundle
+- Use CSS layout, utility, and component classes
+- Render Preact components from our library
+
+## Getting started
+
+1. Open `index.html` in a browser.
```

Comment:
```suggestion
1. Run `./copy.sh`
2. Open `index.html` in a browser.
```

Reply:
My plan was to go back after we publish a version and remove this script, but I did not sufficiently document how this works in the meantime